dev/fattest.simplicity/src/componenttest/rules/repeater/FeatureReplacementAction.java

```diff
@@ -562,89 +709,117 @@ public void setup() throws Exception {
+                    Log.info(c, m, "Failed to create jvm.options in: " + serverName);
+                optionsFound.add(jvmOptionsCreated);
             }
+            serverOptions.addAll(optionsFound);
         }
-        if (replaceFeature != null) {
-            Log.info(c, methodName, "Replace feature [ " + originalFeature + " ] with [ " + replaceFeature + " ]");
-            return replaceFeature;
-        }
-        // We need to check that the feature passed is an EE7/EE8 feature which could have a name change on EE9 that doesn't match
-        // the original feature name it replaces. We also check vice versa if the feature is an EE9 feature that involves a name change
-        // to update from EE9 to EE7/EE8
-        Log.info(c, methodName, "No feature replacement found for [ " + originalFeature + " ]. Verifying if feature name was changed on EE9.");
-        // Reset base feature to not include the "-"
-        baseFeature = baseFeature.substring(0, dashOffset);
-        String featureReplacement = null;
-        for (Map.Entry<String, String> nameChangeEntry : featuresWithNameChangeOnEE9.entrySet()) {
-            if (nameChangeEntry.getValue().equalsIgnoreCase(baseFeature)) {
-                featureReplacement = nameChangeEntry.getKey();
-                Log.info(c, methodName, "Replace EE9 feature [ " + baseFeature + " ] with feature [ " + featureReplacement + " ]");
-                baseFeature = featureReplacement;
-                break;
-            }
-            if (nameChangeEntry.getKey().equalsIgnoreCase(baseFeature)) {
-                featureReplacement = nameChangeEntry.getValue();
-                Log.info(c, methodName, "Replace base feature [ " + baseFeature + " ] with EE9 feature [ " + featureReplacement + " ]");
-                baseFeature = featureReplacement;
-                break;
+
+        Set<File> clientOptions = new HashSet<>();
+        Set<String> clients = new HashSet<>(); // All clients found
+        File clientFolder = new File(pathToAutoFVTTestClients);
+        // Find all jvm.options in the clients
+        clientOptions.addAll(findFile(clientFolder, "jvm.options"));
+        if (clientFolder.exists()) {
+            for (File f : clientFolder.listFiles()) {
+                if (f.isDirectory()) {
+                    clients.add(f.getName());
+                }
             }
         }
-        if (featureReplacement == null) {
-            Log.info(c, methodName, "Remove feature [ " + originalFeature + " ]: No replacement is available");
-            return null;
+
+        // Go through all the clients
+        for (String clientName : clients) {
+            Set<File> optionsFound = findFile(new File(pathToAutoFVTTestServers + clientName), "jvm.options");
+            // If options file doesn't exist
+            if (optionsFound.isEmpty()) {
+                // Create it
+                File jvmOptionsCreated = new File(pathToAutoFVTTestServers + clientName + "/jvm.options");
```

Comment:
Again it should be `pathToAutoFVTTestClients` instead of `pathToAutoFVTTestServers`